compiler/wasm/package.json

```diff
@@ -24,22 +28,53 @@
     "url": "https://github.com/noir-lang/noir/issues"
   },
   "scripts": {
-    "build": "bash ./build.sh",
-    "test": "yarn test:node && yarn test:browser",
-    "test:node": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\"}' mocha",
-    "test:browser": "web-test-runner",
-    "clean": "chmod u+w web nodejs || true && rm -rf ./nodejs ./web ./target ./result",
+    "build": "webpack",
+    "test": "yarn test:build_fixtures && yarn test:node",
+    "test:build_fixtures": "cd test/fixtures && cd simple && nargo compile && cd ../with-deps && nargo compile && cd ../../",
+    "test:node": "NODE_NO_WARNINGS=1 $(yarn bin mocha) --config ./.mocharc.json",
```

Comment:
Why `$(yarn bin mocha)`?

Reply:
Leftover from different tests, restored to just mocha